apps/frontend/src/hooks/aave/useAaveEModeCategories.tsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from 'react';
+
+import { getProvider } from '@sovryn/ethers-provider';
+
+import { BOB_CHAIN_ID } from '../../config/chains';
+
+import { config } from '../../constants/aave';
+import { EModeCategory } from '../../types/aave';
+import { AaveEModeCategories } from '../../utils/aave/AaveEModeCategories';
+import { useAaveReservesData } from './useAaveReservesData';
+
+const eModeCategoriesFetcher = new AaveEModeCategories(
+  config.PoolAddress,
+  getProvider(BOB_CHAIN_ID),
+);
+
+export const useAaveEModeCategories = (): EModeCategory[] => {
+  const { reserves } = useAaveReservesData();
+  const [categories, setCategories] = useState<EModeCategory[]>([]);
+
+  useEffect(() => {
+    eModeCategoriesFetcher
+      .getAllEModeCategories(reserves)
+      .then(setCategories)
+      .catch(console.error);
```

Comment:
should be notify error as I saw above?

Reply:
I mean, we can, but it's like a loading issue, not a problem with user behavior. In practice should not happen other than dev issues, what do you think?